Formula/gosec.rb

```diff
@@ -0,0 +1,33 @@
+class Gosec < Formula
+  desc "Golang security checker"
+  homepage "https://securego.io/"
+  url "https://github.com/securego/gosec/archive/v2.5.0.tar.gz"
+  sha256 "5d4967570f953c8478a3e51a0dee5cfe6313f2570eca15e6fabd5d6379e530f4"
+  license "Apache-2.0"
+  head "https://github.com/securego/gosec.git"
+
+  depends_on "go" => :build
```

Comment:
```suggestion
  depends_on "go"
```

Apparently go is needed at runtime too.

Reply:
yeah, that sounds about right! this is a tool for golang. :D